sys/net/gnrc/network_layer/ipv6/nib/_nib-internal.h

```diff
@@ -22,10 +22,14 @@
 #include <stdbool.h>
 #include <stdint.h>
 
+#include "evtimer_msg.h"
 #include "kernel_types.h"
 #include "mutex.h"
 #include "net/eui64.h"
 #include "net/ipv6/addr.h"
+#ifdef MODULE_GNRC_IPV6
```

Comment:
Are you trolling me? You told me to do so in https://github.com/RIOT-OS/RIOT/pull/6988#discussion_r121294226.